src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    'chip version': version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return 'name: ' + this.name
+        + ', chip version: ' + this['chip version']
+        + ', wheels: ' + wheels;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+
+    goForward(step = 1) {
+      if (step < 1 || typeof step !== 'number') {
+        return this;
+      }
+      this.coords.y += step;
```

Comment:
```suggestion
      if (step > 0) {
        this.coords.y += step;
      }
      
      return this;
```
this will be enough
fix everywhere